### ISSUE TITLE
Startup Directory not respected for cmder tasks

### ISSUE THREAD

**Issue by swimmadude66** [cmderdev/cmder#1648]
OS: Windows 10
cmder Version: 1.3.4

Issue: When I pass a startup dir to cmder consoles, it is ignored. Instead the console always starts up in %CMDER_START. I've tested with other shells and they work fine. I have to assume the arg is being overwritten somewhere in the init.bat.

Sample Task which ignores startupDir:
```
> -cur_console:d:C:\Repos -cur_console:C:%CMDER_ROOT%\icons\cmder.ico cmd /k "%ConEmuDir%\..\init.bat"
```

**Comment by Tchanove** [comment]
Hi @swimmadude66,

This problem seems to be fixed in v1.3.5, give it a try : https://github.com/cmderdev/cmder/releases/tag/v1.3.5

**Comment by swimmadude66** [comment]
Y'all rock, works perfectly!
